task/deploy-gemfab.ts

```diff
@@ -3,11 +3,11 @@ const debug = require('debug')('gemfab:task')
 const dpack = require('@etherpacks/dpack')
 
 const { task } = require('hardhat/config')
-const GASLIMIT = '1000000000000'
 
 task('deploy-gemfab', 'deploy GemFab')
   .addFlag('stdout', 'print the dpack to stdout')
   .addOptionalParam('outfile', 'save the dpack to this path')
+  .addOptionalParam('gasLimit', 'gemfab deploy tx gas limit')
```

Comment:
doesn't look like it handles not providing gasLimit so could be addParam instead of addOptionalParam

Reply:
it works without a gaslimit

Reply:
this is just good in case gas estimation gets wonky which it sometimes does